### DISCUSSION TITLE
sync changes done outside NocoDB [Jul 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2996] [Discord Threads]
> 🙋‍♂️ **pyther**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@573626738349965312> You will have to trigger meta sync explicitly. Details here    
#### https://docs.nocodb.com/setup-and-usages/sync-schema#how-to-sync-schema-changes-to-nocodb  
#  
> 👨‍🦲 **dstala**  
#### You will have to sync schema changes explicitly. Details here    
#### https://docs.nocodb.com/setup-and-usages/sync-schema#how-to-sync-schema-changes-to-nocodb  
#  
> 🙋‍♂️ **pyther**  
#### Thanks! I just tried and I got a "request failed with status code 400".  
#  
> 🙋‍♂️ **pyther**  
#### <@857998597474484295> Does the connection need certain permissions? Currently, the connection I have is a read only permissions from the database.  
#  
> 💁 **o1lab**  
#### > request failed with status code 400   
#### are you seeing this after hitting Sync NOw button ?  
#  
> 💁 **o1lab**  
#### can u pls see the errors in docker console  
#  
> 🙋‍♂️ **pyther**  
#### Yes, here's the message I see when the request runs  
![image0](https://cdn.discordapp.com/attachments/1001539586759073972/1001547883088191649/unknown.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1001539586759073972)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
